lib/.jshintrc

```diff
@@ -1,21 +1,21 @@
 {
-  "curly": false,
-  "eqeqeq": true,
-  "immed": true,
-  "latedef": true,
-  "newcap": true,
-  "noarg": true,
-  "sub": true,
-  "undef": true,
-  "boss": true,
-  "eqnull": true,
-  "es3": true,
-  "notypeof": true,
-  "globals": {
-    "console": false,
-    "exports": false,
-    "define": false,
-    "module": false,
-    "window": false
-  }
+    "curly": false,
+    "eqeqeq": true,
+    "esversion": "6",
+    "immed": true,
+    "latedef": true,
+    "newcap": true,
+    "noarg": true,
+    "sub": true,
+    "undef": true,
+    "boss": true,
+    "eqnull": true,
+    "notypeof": true,
+    "globals": {
+        "console": false,
+        "exports": false,
+        "define": false,
+        "module": false,
+        "window": false
+    }
```

Comment:
Minor nit: this indentation change [contravenes `.editorconfig`](https://github.com/pimterry/loglevel/blob/7a4b1d963154874f6035338b46d269c40e5d2761/.editorconfig#L20-L21). Either this or that should change so they match up.